scripts/utils.sh

```diff
@@ -2,7 +2,7 @@
 
 # Functions
 fn_die() {
-  echo -e "\n\033[1m${1}\033[0m\n" >&2
+  echo -e "\n\033[0;31m\033[1m${1}\033[0m\n" >&2
```

Comment:
shouldn't this be something like: 
echo -e "\n\033[0;1;31m${1}\033[0m\n" >&2

Reply:
This will produce BOLD RED color for echo message under fn_die function :shrug:

Reply:
Try it 
```
$ echo -e "\n\033[0;31m\033[1mERROR MESSAGE\033[0m\n" >&2
```